### PR TITLE
Reformat package-lock.json and package.json to single spacing.

### LINKED ISSUE #1552: Change workflow check to reformate package-lock.json
**Describe the bug**

Dependabot is producing files that reformat the file that conflicts with the teams style rules.

**Version**

What is the version of Horreum ?

```
0.12.0
```

### PR DESCRIPTION
**Pull request by whitingjr**
## Fixes Issue

fixes #1552 

## Changes proposed

[x] - Change workflow to write the file in the correct format

### CODE REVIEW

**Comment by stalep** [comment]
I'm not comfortable merging this without having John looking at it first. Let's put this on hold until he is back.

**Comment by johnaohara** [comment]
was searching for this issue and came across this  comment: https://github.com/npm/json-parse-even-better-errors/issues/3#issuecomment-1668728929

` Maybe those people complaining about their package-lock not using ... double-spacing just like their package.json file were just a vocal minority, ` 

our package.json is double spaced

We had a brief discussion about trying;
- single spacing out package.json
- fixing the npm version in package.json to ensure dependabot picks the correct version: https://stackoverflow.com/questions/29349684/how-can-i-specify-the-required-node-js-version-in-package-json
- explicitly defining the npm version in https://github.com/Hyperfoil/Horreum/blob/master/.github/dependabot.yml

**Comment by whitingjr** [comment]
Setting versions too version npm 10 and node 20.

File: horreum-web/package.json
Comment (johnaohara) [inline horreum-web/package.json:1]: in order for later versions of npm to not revert `horreum-web/package-lock.json` to double spaced, this file needs to be single spaced as well

Comment (whitingjr) [reply horreum-web/package.json:1]: thanks for spotting that.

**Comment by whitingjr** [comment]
@johnaohara  Both package.json and package-lock.json are re-formatted in this commit.

**Comment by johnaohara** [comment]
@whitingjr this looks great, thanks for fixing this issue

**Pull request merged**